File: track-api/Models/Db/Series.cs
namespace track_api.Models.Db
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Series
    {
        public Series()
        {
            Visible = true;
        }

        public int Id { get; set; }

        public int DatasetId { get; set; }

        public int TypeId { get; set; }

        public string Label { get; set; }

        [StringLength(6)]
        public string Color { get; set; }

        public string Unit { get; set; }

        public int Order { get; set; }

        public bool Visible { get; set; }
    }
}

File: track-web/src/models/odata/Series.ts
export class Series {

  public Id: number;
  public DatasetId: number

  public TypeId: number;
  public Label: string;
  public Color: string;

  public Order: number;
  public Visible: boolean;

  public static Default = (id: number = 0): Series => {
    return {
      Id: id,
      Label: '',
      TypeId: 2,  // Decimal
      Visible: true,
      Order: id
    } as Series;
  }
}